typegate/src/planner/mod.ts

```diff
@@ -470,4 +493,35 @@ export class Planner {
   private formatPath(path: string[]) {
     return [this.operationName, ...path].join(".");
   }
+
+  /**
+   * Idea:
+   * query: [a1, a2, ..], mutation: [b1, a1, b2, ..]
+   * => a1: [query, mutation], a2: [query], ..., b1: [mutation] ...
+   */
+  private getReverseMapNameToQuery() {
+    const indices = ["query", "mutation"].map((label) =>
```

Comment:
I would generalize this to any types included in the `properties` of the root type. We may support some subscription in the future or even go beyond what graphql spec recommend.